src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -180,6 +217,12 @@ def description_wrapper(description)
     end
   end
 
+  def webui2_description_wrapper(description)
```

Comment:
Why do you do this? Does `plain` even have any effect? And can't you get the same without this helper?

Reply:
Ok, I will remove it and do that in the view